UefiPayloadPkg/PayloadLoaderPeim/PayloadLoaderPeim.inf

```diff
@@ -52,13 +52,32 @@
 
 [Ppis]
   gEfiPeiLoadFilePpiGuid                 ## PRODUCES
+  gEfiEndOfPeiSignalPpiGuid              ## CONSUMES
+  gEfiReadyToPayloadPpiGuid              ## PRODUCES

```

Comment:
1. Please remove Efi prefix for gEfiReadyToPayloadPpiGuid.
2. see if we could install this PPI after gEfiEndOfPeiSignalPpi to ensure sequence.